pkgs/development/python-modules/rdkit/default.nix

```diff
@@ -51,26 +51,27 @@ buildPythonPackage rec {
       owner = pname;
       repo = pname;
       rev = "Release_${versionTag}";
-      sha256 = "19idgilabh04cbr1qj6zgrgsfjm248mmfz6fsr0smrd68d0xnml9";
+      hash = "sha256-AaawjCv3/ShByOKU0c37/hjuyfD7NhFC8UngDoG7C0s=";
     };
 
   unpackPhase = ''
     mkdir -p source/External/AvalonTools/avalon source/External/YAeHMOP/yaehmop source/External/FreeSASA/freesasa
     cp -r ${src}/* source
-    cp -r ${external.avalon}/SourceDistribution/* source/External/AvalonTools/avalon
+    cp -r ${external.avalon}/* source/External/AvalonTools/avalon
     cp -r ${external.yaehmop}/* source/External/YAeHMOP/yaehmop
     cp -r ${external.freesasa}/* source/External/FreeSASA/freesasa
```

Comment:
Those must be copied?

Reply:
Thanks for the push on this. I changed to link `avalon` and `yaehmop` instead of copying (`freesasa` must still be copied because the source tree is modified), and tried to generally clean up the `unpackPhase`.